drivers/media/platform/bcm2835/bcm2835-camera.c

```diff
@@ -485,18 +485,18 @@ static void buffer_queue(struct vb2_buffer *vb)
 		v4l2_err(&dev->v4l2_dev, "%s: error submitting buffer\n",
 			 __func__);
 
-    if (dev->capture.frame_count && is_capturing(dev) && empty) {
-        v4l2_dbg(1, bcm2835_v4l2_debug, &dev->v4l2_dev,
-                 "Grab a frame on buffer filled\n");
-        vchiq_mmal_port_parameter_set(
-            dev->instance,
-            dev->capture.
-            camera_port,
-            MMAL_PARAMETER_CAPTURE,
-            &dev->capture.
-            frame_count,
-            sizeof(dev->capture.frame_count));
-    }
+	if (dev->capture.frame_count && is_capturing(dev) && empty) {
+		v4l2_dbg(1, bcm2835_v4l2_debug, &dev->v4l2_dev,
+				 "Grab a frame on buffer filled\n");
```

Comment:
Seeing as github complains that I commented on an outdated diff:
"Grab a frame on buffer queued", not "filled".